.changeset/light-melons-share.md

```diff
@@ -0,0 +1,11 @@
+---
+'@roadiehq/backstage-plugin-prometheus': major
```

Comment:
This doesn't look like a breaking change so a `minor` bump is enough when introducing new functionality I think

Reply:
Good point :+1:. Fixed thank you